src/api/TokenApi/tokenList.ts

```diff
@@ -14,21 +14,21 @@ const tokens: TokenDetailsByNetwork[] = [
     symbol: 'WETH',
     decimals: 18,
     addressByNetwork: {
-      1: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
-      4: '0xc778417e063141139fce010982780140aa0cd5ab',
+      [Network.Mainnet]: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+      [Network.Rinkeby]: '0xc778417e063141139fce010982780140aa0cd5ab',
     },
   },
 
   // Tether:
   //  https://tether.to/
-  //  Fiat enabled collateralized stable coin, that is backed by the most popular fiat currency, USD (US Dollar) in a 1:1 ratio
+  //  Fiat enabled collateralized stable coin, that is backed by the most popular fiat currency, USD (US Dollar) in a [Network.Mainnet]:1 ratio
```

Comment:
You might not want to update this comment :)

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FoCCLHVNt8YO64%2Fgiphy.gif&token=copy" data-gitmeme-token="copy"><img src="https://media2.giphy.com/media/oCCLHVNt8YO64/giphy.gif" title="Created by gitme.me with /copy"/></a> paste 
Ups...

Thanks